.github/workflows/build.yml

```diff
@@ -12,6 +12,9 @@ on:
 
 env:
   CACHE_VERSION: 1
+  Z3_VERSION: "4.8.10"
```

Comment:
While I'm thinking about it, there are a couple of places elsewhere in the repo where we hardcode this version number, such as in the various Dockerfiles (see [this commit](https://github.com/GaloisInc/saw-script/commit/68f6add5a32b2067ddf3464680acd2c365c189a8#diff-40506c1c0ab9feaab92a3c2136415b1def6045b45abfcbf047211e24d4be39b3)). I'm not sure if it's practical to design things in such a way that this version number is only ever declared once, so we should leave comments indicating that if you update the version number in one place, you should update them elsewhere.

Similar comments apply to `CVC4_VERSION` and `YICES_VERSION` below.

Reply:
> I'm not sure if it's practical to design things in such a way that this version number is only ever declared once

There may be some hope for this yet - @atomb and I have discussed creating a docker baselayer image containing a distribution of solvers (from which the binaries could be extracted for CI). Agree that a comment would be helpful in its current state - I've added one.